qabelbox/src/main/java/de/qabel/qabelbox/chat/view/views/ChatFragment.kt

```diff
@@ -142,6 +147,19 @@ class ChatFragment : ChatView, BaseFragment(), AnkoLogger {
         }
 
         bt_send.onClick { presenter.sendMessage() }
+        deactivateSendButton()
```

Comment:
Extract the onTextChanged handler method and just call it here. This makes future modifications a bit harder, e.g. restoring the last unsent message into the message field.